lang/warns.json

```diff
@@ -0,0 +1,26 @@
+{
+  "en": {
+    "help": "This module can warn a person\nAvailable commands:\n/max_warns <num> - maximum amount warn\n/warn - reply to the person\n/unwarn - reply to the person",
```

Comment:
You can write smth like this: `Reply to somebody's message with /warn to warn him/her`